tests/test-values.yaml

```diff
@@ -497,3 +497,9 @@ forceSchemaArrayModifier:
   - w: "x"
     y:
       - "z"
+
+## Invented parameter to test proper modifier consumption
+## @param modifierWithSquareBrackets [object] Additional square brackets should appeare in description. [More information here](#additional-square-brackets).
+modifierWithSquareBrackets:
```

Comment:
All the modifiers are applied using square brackets so this name seems a bit confusing, could you rename it to something like `descriptionWithLink` or `linkInDescription` or anything you prefer describes better the test case please?

Reply:
Changed the parameter to `linkInDescription`